code-studio/src/js/progress.js

```diff
@@ -56,9 +64,19 @@ progress.populateProgress = function (scriptName, puzzlePage) {
     // Merge progress from server (loaded via AJAX)
     var serverProgress = data.levels || {};
     Object.keys(serverProgress).forEach(function (levelId) {
-      // Only the server can speak to whether a level is submitted.  If it is,
-      // we show the submitted styling.
-      if (serverProgress[levelId].submitted) {
+      // Only the server can speak to whether a level is submitted/accepted/rejected.  If it is,
```

Comment:
During HOC when we are in high scale mode, what will this look like? Will the call to get progress for these dots be allowed?